tuned/plugins/plugin_vm.py

```diff
@@ -119,3 +141,84 @@ def _get_transparent_hugepage_defrag(self):
 			return cmd.get_active_option(cmd.read_file(sys_file))
 		else:
 			return None
+
+	def _check_twice_pagesize(self, option, int_value):
+		min_bytes = 2 * int(cmd.getconf("PAGESIZE"))
+		if int_value >= min_bytes:
+			return True
+		log.error("The value of '%s' must be at least twice the page size (%s)." % (option, min_bytes))
```

Comment:
Nit: This is a matter of opinion/style, but I typically tend to do the "[Happy path](https://en.wikipedia.org/wiki/Happy_path)".  I.e. I deal with issues (like returning errors first) and then continue to solve the problem (here return True) in the end.  Similarly elsewhere in the code.  Not a deal breaker or a reason not to merge this though.

Reply:
To be honest, that sounds like a nice practice... Updated.